README.md

```diff
@@ -54,11 +54,11 @@ Each module will also have different minimum variables that need to be set.
 Ex.
 ```
 $ curl -sSL https://zipkin.io/quickstart.sh | bash -s
-$ curl -sSL https://zipkin.io/quickstart.sh | bash -s io.zipkin.gcp:zipkin-module-storage-stackdriver:LATEST:module stackdriver.jar
+$ curl -sSL https://zipkin.io/quickstart.sh | bash -s io.zipkin.gcp:zipkin-module-gcp:LATEST:module gcp.jar
```

Comment:
will probably release this as version 1.0 vs perpetual <1.0, and doing this rename now is the best time to do it